src/test/App.test.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Enzyme, { mount } from "enzyme";
+import App, { makeCards } from "../App";
+import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
+import { CARD_DATA } from "../card-constants";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<App />", () => {
+  it("renders <App /> component", () => {
+    const wrapper = mount(<App />);
+    expect(wrapper.find(App)).toHaveLength(1);
+  });
+
+  it("generates an array of cards", () => {
+    let result = makeCards(CARD_DATA);
```

Comment:
Does the `makeCards` function "shuffle" the `CARD_DATA`? if so, I would assert that it's doing the shuffling.